lucky_draw/lib/lucky_draw_web/schema.ex

```diff
@@ -0,0 +1,13 @@
+defmodule LuckyDrawWeb.Schema do
+  use Absinthe.Schema
+  import_types(LuckyDrawWeb.Schema.Lottery)
+
+  alias LuckyDrawWeb.Resolver
+
+  query do
+    @desc "Get all awards"
+    field :awards, list_of(:award) do
+      resolve(&Resolver.Lottery.list_awards/3)
```

Comment:
learned something, cool
_reference to the 3 arity function all_links found in the CommunityWeb.NewsResolver module_

originally this is weird to me til I search for it.